app/src/main/res/xml/preferences_alarms.xml

```diff
@@ -8,6 +8,11 @@
         android:title="Enable Alarms"
         android:summary="Allow the phone to vibrate as a warning" />
 
```

Comment:
I'm not sure that it is good idea to put inmotion-specific option to general menu. Maybe it's better to move it to inmotion_preferences, or at least make it disabled by default?

Reply:
You`re right. I`ll move it.